Casks/p/privatevpn.rb

```diff
@@ -9,7 +9,7 @@
 
   livecheck do
     url "https://privatevpn.com/why-privatevpn/view-our-software/"
```

Comment:
```suggestion
    url "https://privatevpn.com/why-privatevpn/view-our-software"
```

Looks like this URL redirects to the same URL without the trailing slash, so we should update this while we're at it.

Reply:
Good catch!